### PR TITLE
Serbian Survivor

### PR DESCRIPTION
**Pull request by DarkLevelSS13**
## About The Pull Request
image
Serbians now will have IH perk "Survivor".

## Changelog
:cl: DarkLevel
balance: Serbians now spawns with SURVIVOR perk
code: Add perks value to antags
/:cl:

### CODE REVIEW

File: code/game/antagonist/outer/mercenary/mercenary.dm [hunk 1]
```diff
@@ -52,6 +52,9 @@
 	for(var/name in stat_modifiers)
 		L.stats.changeStat(name, stat_modifiers[name])
 
+	for(var/perk in perks)
```
Comment (TheShown911) [inline code/game/antagonist/outer/mercenary/mercenary.dm:55]: I'd give this to base antagonist equip()

**Review by TheShown911 — APPROVED** [review]

File: code/game/antagonist/outer/mercenary/mercenary.dm (hunk 1 above)
Comment (TorinoFermic) [reply code/game/antagonist/outer/mercenary/mercenary.dm:55]: Yes, this new code should be in base antag equipd()

Comment (DarkLevelSS13) [reply code/game/antagonist/outer/mercenary/mercenary.dm:55]: Do you see a base antagonist equip()?
Base antagonist equip() don't have even stat_modifiers

Comment (DarkLevelSS13) [reply code/game/antagonist/outer/mercenary/mercenary.dm:55]: What if it's blitzshell or borer?

Comment (DarkLevelSS13) [reply code/game/antagonist/outer/mercenary/mercenary.dm:55]: I resolve this review, because of the build code.

Comment (TorinoFermic) [reply code/game/antagonist/outer/mercenary/mercenary.dm:55]: They are in code\game\antagonist\antagonist_equip.dm at first line

Comment (DarkLevelSS13) [reply code/game/antagonist/outer/mercenary/mercenary.dm:55]: I know where parent proc is, i say that if i do this there will be bunch of runtimes.
There are a reason why stats not in parent proc. What about borer or blitzhell? They don't have stats.
What if somebody want to add their no-human antag? More runtimes.
No, i don't want that.

**Review by mattroks101 — APPROVED** [review]
This will do for now. A refactor of the antag perks will have to come later.

**Pull request merged**